packages/core-utils/src/types.js

```diff
@@ -511,7 +511,10 @@ export const queryType = PropTypes.shape({
   startTime: PropTypes.string,
   endTime: PropTypes.string,
   mode: PropTypes.string,
-  showIntermediateStops: PropTypes.bool,
+  showIntermediateStops: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.string
+  ]),
```

Comment:
This doesn't seem like the right approach. Can we not change otp-rr to correctly set the value as boolean?

Reply:
@evansiroky or @landonreed do you know the rationale for passing `"true"` and `"false"` as a string in this test file? https://github.com/opentripplanner/otp-ui/blob/master/packages/core-utils/src/__tests__/query.js#L14

Two things I can do regardless:
- Convert all the "boolean" fields to just `String`,
- Or, just remove all fields that are not relevant to `SettingsSelectorPanel`.

Reply:
@binh-dam-ibigroup, the input params in this test file are designed to mimic what comes from directly from the URL query params (i.e., the output of [`#getUrlParams`](https://github.com/ibi-group/otp-ui/blob/update-leaflet-for-mapbox/packages/core-utils/src/query.js#L62-L64)). I think the correct approach is to just change `planParamsToQuery` to parse booleans correctly or somehow handle this with `query-params.js`.

Reply:
@evansiroky what do you think? Dealing with `planParamsToQuery` or `query-params` feels more than a UI tweak.

Reply:
@landonreed and I just agreed to remove `queryType` altogether.

For context, `queryType` was only used to avoid placing a lint PropType exception for `SettingsSelectorPanel` and `GeneralSettingsPanel` in package `trip-form`. Neither control uses any of the type members explicitly (the controls get which parameters to display from `planParamsToQuery` and `query-params`).